scripts/quests/windurst/Toraimarai_Turmoil.lua

```diff
@@ -0,0 +1,142 @@
+-----------------------------------
+-- Toraimarai Turmoil
+-----------------------------------
+-- !addquest 2 80
+-- Ohbiru-Dohbiru : !pos 23 -5 -193 238
+-- Leepe-Hoppe    ! !pos -131 20 -174 238
+-- Polikal-Ramikal: !pos 15 -18 195 239
+-- Yoran-Oran     : !pos -110 -14 203 239
+-- Giddeus Spring : !pos -258 -2 -249 145
+-----------------------------------
+
+local quest = Quest:new(xi.questLog.WINDURST, xi.quest.id.windurst.TORAIMARAI_TURMOIL)
+
+quest.reward =
+{
+    gil  = 4500,
+    fame = 100,
+    fameArea = xi.fameArea.WINDURST,
+    title = xi.title.CERTIFIED_RHINOSTERY_VENTURER
```

Comment:
Small thing, but could you please add a trailing comma to the last entry as well?

Reply:
Sure thing 👍